tests/data/test.geojson

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
I would prefer we don't hard-code any test data files. Let's use a Rust crate like `geojson` to generate these files in `TempDir`.

Reply:
Done